library/clojure

```diff
@@ -1,6 +1,7 @@
 # maintainer: Paul Lam <paul@quantisan.com> (@Quantisan)
 
-latest: git://github.com/Quantisan/docker-clojure@27d0141c22bedbf78b908bf5bebd6629fc10a491
+latest: git://github.com/Quantisan/docker-clojure@5c988033c6510c7d91a5f6c7a91b4f63f8f90768
 
+lein-2.5.1: git://github.com/Quantisan/docker-clojure@5c988033c6510c7d91a5f6c7a91b4f63f8f90768
 lein-2.5.0: git://github.com/Quantisan/docker-clojure@27d0141c22bedbf78b908bf5bebd6629fc10a491
```

Comment:
You don't need to keep 2.5.0 in here, do you? (I mean, you don't plan to update it later if 2.5.0 has a problem -- you'd just update 2.5.1, right?)

We won't delete the tag on the Hub, but then it'll be removed from the "Supported" list on the main image description.